python/requirements/ml/requirements_dl.txt

```
@@ -13,3 +13,9 @@ torchvision==0.10.0;sys_platform=="darwin"
 -f https://download.pytorch.org/whl/torch_stable.html
 torch==1.9.0+cpu;sys_platform!="darwin"
 torchvision==0.10.0+cpu;sys_platform!="darwin"
+
+jax==0.1.69;sys_platform!="darwin"
+jaxlib==0.1.69;sys_platform!="darwin"
+flax==0.4.0;sys_platform!="darwin"
+optax==0.0.9;sys_platform!="darwin"
+einops==0.4.1;sys_platform!="darwin"
```

```suggestion
jax==0.1.69
jaxlib==0.1.69
flax==0.4.0
optax==0.0.9
einops==0.4.1
```

to be able to run on CPU Mac?

maybe also with a 'please sync versions' note like on L6?